composer.json

```diff
@@ -16,7 +16,7 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Interop\\Async\\Loop\\": "test"
+            "Interop\\Async\\Loop\\Test\\": "test"
```

Comment:
event-loop-tests currently provides one single class Interop\Async\Loop\Test.

But still, alone because of the DummyDriver it ought to not be in the main Loop namespace.